Description: Adds an explicit dependency on JavaFX to build with Java 9+
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/engine/pom.xml
+++ b/engine/pom.xml
@@ -48,6 +48,12 @@
         Provided dependencies
         -->
         <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-base</artifactId>
+            <version>debian</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
             <groupId>javax.el</groupId>
             <artifactId>javax.el-api</artifactId>
             <scope>provided</scope>
